benches/bench.rs

```diff
@@ -89,6 +89,68 @@ mod poly_benches {
     }
 }
 
+mod commitment_benches {
+    use super::*;
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
+
+    /// Benchmarks [de]ser of a univariate commitment
+    fn roundtrip(c: &mut Criterion) {
+        let mut rng = XorShiftRng::from_seed(RNG_SEED);
+        c.bench_function_over_inputs(
+            "Commitment [de]Serializiation",
+            move |b, &&deg| {
+                let rand_factors = || {
+                    let poly = Poly::random(deg, &mut rng).commitment();
+                    let poly_ser = bincode::serialize(&poly).expect("lhs boom ser");
+                    poly_ser
+                };
+                b.iter_with_setup(rand_factors, |p| {
+                    bincode::deserialize::<Commitment>(&p).expect("lhs boom deser")
+                })
+            },
+            &TEST_DEGREES,
+        );
+    }
+
+    criterion_group! {
+        name = commitment_benches;
+        config = Criterion::default();
+        targets = roundtrip,
+    }
+}
+
+mod bicommitment_benches {
+    use super::*;
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
+
+    /// Benchmarks [de]ser of a univariate commitment
+    fn roundtrip(c: &mut Criterion) {
+        let mut rng = XorShiftRng::from_seed(RNG_SEED);
+        c.bench_function_over_inputs(
+            "BiCommitment [de]Serializiation",
```

Comment:
```suggestion
            "BiCommitment Deserializiation",
```
Same comment as above